script/BaseScript.s.sol

```diff
@@ -54,30 +52,22 @@ struct BaseInput {
 struct Deployment {
     ChainDeployment[] chains;
     address deployerAddress;
-    string erc20Name;
-    string erc20Symbol;
 }
 
 struct ChainDeployment {
     uint256 chainId;
     address erc20Address;
     address erc20ProxyAdmin;
+    address erc20Implementation;
     bool isL1;
     address lzEndpoint;
     uint32 lzEID;
-    address multiChainDeployer;
     address oftAdapter;
     address oftAdapterProxyAdmin;
+    address oftAdapterImplementation;
     address oftAdapterTimelock;
 }
 
-struct PredictedAddresses {
```

Comment:
predicted addresses for other networks is no longer required, we just `CREATE3Factory.getDeployer(address deployer,bytes32 salt)` directly

Reply:
simple